include/asset_manager.hpp

```diff
@@ -96,6 +96,7 @@ namespace InputManager
     {
     public:
         virtual void mouseCallback(GLFWwindow*, double, double) = 0;
```

Comment:
can you help me understand the asset manager abstraction. I've previously seen methods like this in a class of event recievers